Makefile

```diff
@@ -41,7 +41,12 @@
 #
 ###############################################################################
 
-get_common:=$(shell wget -nv https://raw.githubusercontent.com/rafaelvanoni/common/master/Makefile.common -O Makefile.common)
+GO_BUILD_VER?=v0.26
+
+MAKE_BRANCH?=$(GO_BUILD_VER)
+MAKE_REPO?=https://raw.githubusercontent.com/projectcalico/go-build/$(MAKE_BRANCH)/Makefile.common
+
+get_common:=$(shell wget -nv $(MAKE_REPO) -O Makefile.common)
 include Makefile.common
```

Comment:
This will get the common Makefile every time.  Make has a nice way of handling this, you can include a file that itself is built by a make target.  You can do something like this to make sure that it only grabs the file if it hasn't already been downloaded:
```
include Makefile.common
MAKE_BRANCH?=$(GO_BUILD_VER)
Makefile.common: Makefile.common.$(MAKE_BRANCH)
	cp "$<" "$@"
Makefile.common.$(MAKE_BRANCH):
	# Clean up any files downloaded from other branches so they don't accumulate.
	rm -f Makefile.common.*
	wget -nv https://raw.githubusercontent.com/projectcalico/go-build/$(MAKE_BRANCH)/Makefile.common -O "$@"
```
The intermediate step to create `Makefile.common.$(MAKE_BRANCH)` ensures that we re-get the file whenever `$(MAKE_BRANCH)` changes.

Reply:
I added `-nc` to the wget call, it has the same effect.

Reply:
I think @fasaxc 's suggestion will also ensure the file is downloaded if the GO_BUILD_VER/MAKE_BRANCH changes but I don't think it would be with the current wget.

Reply:
Fixed